pkg/bpf/detect_linux.go

```diff
@@ -231,6 +232,37 @@ func detectLSM() bool {
 	if features.HaveProgramType(ebpf.LSM) != nil {
 		return false
 	}
+	_, err := os.Stat("/sys/kernel/security")
+	if err != nil {
+		if os.IsNotExist(err) {
+			if err := os.Mkdir("/sys/kernel/security", 0755); err != nil {
```

Comment:
is this needed? IIUC the `/sys/kernel/security` directory should be always there (I checked 4.19)
also I wonder we'd be allowed to create that directory anyway